recipe-server/client/control/components/RecipeForm.js

```diff
@@ -215,6 +225,18 @@ export function initialValuesWrapper(Component) {
     if (location.state && location.state.selectedRevision) {
       initialValues = location.state.selectedRevision;
     }
+
+    // If we still don't have initial values, roll with the defaults.
+    if (!initialValues) {
+      initialValues = Object.assign({}, DEFAULT_FORM_VALUES);
```

Comment:
The front end uses the spread operator instead of `Object.assign`, ex:

`initalValues = { ...DEFAULT_FORM_VALUES };`